docs/platforms/aws-lambda/nodejs/index.md

```diff
@@ -0,0 +1,465 @@
+
+aws.sqs.queue_name: "MyQueue" // Taken from the Queue URL
+aws.sqs.operation: "send" // Must be "send"
+aws.sqs.message_ids: ["fja98jafs"] // An array with the message ID provided in the SDK operation response.  Must be an array.
+```
+
+### `aws.sdk.sqs.sendmessagebatch`
+
+If you use the `aws-sdk` module in Node.js with AWS SQS and perform a `SendMessageBatch` operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.sqs */
+
+aws.sqs.queue_name: "MyQueue" // Taken from the Queue URL
+aws.sqs.operation: "send" // Must be "send"
+aws.sqs.message_ids: ["fja98jafs", ...] // An array with the message IDs provided in the SDK operation response.
+```
+
+### `aws.sdk.sns.publish`
+
+If you use the `aws-sdk` module in Node.js with AWS SNS and perform a `Publish` operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.sns */
+
+aws.sns.topic.name: "sns-lambda" // Taken from the TopicArn
+aws.sns.operation: "send" // Must be "send"
+aws.sns.message_ids: ["fja98jafs"] // Introspected from the response of this SDK operation
+```
+
+### `aws.sdk.sns.publishbatch`
+
+If you use the `aws-sdk` module in Node.js with AWS SNS and perform a `PublishBatch` operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.sns */
+
+aws.sns.topic.name: "sns-lambda" // Taken from the TopicArn
+aws.sns.operation: "send" // Must be "send"
+aws.sns.message_ids: ["fja98jafs", ...] // Introspected from the response of this SDK operation
+```
+
+## `node.http`
+
+If you use the `http` module in Node.js, this Span is created.
+
+### Tags
+
+These are the Tags attached to this Span:
+
+```javascript
+/* Tags: Standard */
+
+s.span.id: "jf0asf90j1jasf" // This Span's ID
+s.span.parent.id: "e8a4e2a264145041" // This Span's parent Span ID.
+s.span.timestamp: "2022-07-13T20:10:48.794945024Z" // This Span's time.  Same as start_time.
+s.span.start_time: "2022-07-13T20:10:48.794945024Z" // This Span's start time.
+s.span.end_time: "2022-07-13T20:10:48.799375104Z" // This Span's end time.
+
+/* Tags: node.http */
+
+node.http.url: "http://myapp.com"
+node.http.method: "get"
+node.http.path: "/helloworld"
+node.http.status_code: 401
+```
+
+## `node.https`
+
+If you use the `https` module in Node.js, this Span is created.
+
+### Tags
+
+These are the Tags attached to this Span:
+
+```javascript
+/* Tags: Standard */
+
+s.span.id: "91901jasjfsaofj" // This Span's ID
+s.span.parent.id: "e8a4e2a264145041" // This Span's parent Span ID.
+s.span.timestamp: "2022-07-13T20:10:48.794945024Z" // This Span's time.  Same as start_time.
+s.span.start_time: "2022-07-13T20:10:48.794945024Z" // This Span's start time.
+s.span.end_time: "2022-07-13T20:10:48.799375104Z" // This Span's end time.
+
+/* Tags: node.https */
+
+node.https.url: "http://myapp.com"
+node.https.method: "get"
+node.https.path: "/helloworld"
+node.https.status_code: 500
+```
+
+## `express`
+
+If an HTTP request is made from your AWS Lambda function, a Span is created.
```

Comment:
`express` feels specific to the Express web application framework. While I believe this span is about the handling of application endpoints (behind HTTP or HTTPS) in general (and that can be configured with many different frameworks).

Wouldn't it be better then to name it as `app` (some name generic to `http` and `https`), and in span property store wether its _http_ or _https_ and if Framework is used, name of the framework

Reply:
This is specific to `express`, and not about handling of application endpoints.  I still think we should not create abstract terms or get cute at this time.  People should be able to look at the name of the span and immediately understand what it is without relying on documentation to decode what it means.  This Span is intended to measure everything of Express.js (init, pre-middleware, request handler, post-middleware, etc.).

Reply:
Though, I do want to acknowledge I think your idea is clever!  Let's keep it simple for now, then figure out where to optimize conveying what is happening in users' logic.